lighthouse-core/test/audits/speed-index-metric-test.js

```diff
@@ -74,7 +74,6 @@ describe('Performance: speed-index-metric audit', () => {
     const artifacts = mockArtifactsWithSpeedlineResult(SpeedlineResult);
 
     return Audit.audit(artifacts).then(response => {
-      assert.equal(response.displayValue, '845');
```

Comment:
If that's true, what is this bit doing? https://github.com/GoogleChrome/lighthouse/blob/3fefe78dd1b4a1b50496152422130e55eeee8cc4/lighthouse-core/audits/audit.js#L60-L63